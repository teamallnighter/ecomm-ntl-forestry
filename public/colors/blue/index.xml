<?xml version="1.0" encoding="utf-8" standalone="yes"?>
<rss version="2.0" xmlns:atom="http://www.w3.org/2005/Atom">
  <channel>
    <title>blue on Adrian Hugo | E-commerce template</title>
    <link>https://examplesite.com/colors/blue/</link>
    <description>Recent content in blue on Adrian Hugo | E-commerce template</description>
    <generator>Hugo -- gohugo.io</generator>
    <language>en-us</language>
    <lastBuildDate>Mon, 25 Nov 2019 13:39:07 +0600</lastBuildDate>
    
	<atom:link href="https://examplesite.com/colors/blue/index.xml" rel="self" type="application/rss+xml" />
    
    
    <item>
      <title>Blue Shirt</title>
      <link>https://examplesite.com/shop/blue-shirt/</link>
      <pubDate>Mon, 25 Nov 2019 13:39:07 +0600</pubDate>
      
      <guid>https://examplesite.com/shop/blue-shirt/</guid>
      <description>Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas. Vestibulum tortor quam, feugiat vitae, ultricies eget, tempor sit amet, ante.</description>
    </item>
    
    <item>
      <title>Floral Kirby</title>
      <link>https://examplesite.com/shop/floral-kirby/</link>
      <pubDate>Mon, 25 Nov 2019 13:39:07 +0600</pubDate>
      
      <guid>https://examplesite.com/shop/floral-kirby/</guid>
      <description>Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas. Vestibulum tortor quam, feugiat vitae, ultricies eget, tempor sit amet, ante.</description>
    </item>
    
    <item>
      <title>Lady Shoe</title>
      <link>https://examplesite.com/shop/lady-shoe/</link>
      <pubDate>Mon, 25 Nov 2019 13:39:07 +0600</pubDate>
      
      <guid>https://examplesite.com/shop/lady-shoe/</guid>
      <description>Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas. Vestibulum tortor quam, feugiat vitae, ultricies eget, tempor sit amet, ante.</description>
    </item>
    
  </channel>
</rss>