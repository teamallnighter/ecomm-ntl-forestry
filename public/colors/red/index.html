<!DOCTYPE html>
<html lang="en-us"><head>
  <meta charset="utf-8">
  <title>Adrian Hugo | E-commerce template</title>

  <!-- mobile responsive meta -->
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
  <meta name="description" content="This is meta description">
  
  <meta name="author" content="Themefisher">
  <meta name="generator" content="Hugo 0.74.3" />

  <!-- plugins -->
  
  <link rel="stylesheet" href="https://examplesite.com/plugins/bootstrap/bootstrap.min.css ">
  
  <link rel="stylesheet" href="https://examplesite.com/plugins/themify-icons/themify-icons.css ">
  
  <link rel="stylesheet" href="https://examplesite.com/plugins/slick/slick.css ">
  
  <link rel="stylesheet" href="https://examplesite.com ">
  
  <link rel="stylesheet" href="https://cdn.snipcart.com/themes/v3.0.3/default/snipcart.css ">
  

  <!-- Main Stylesheet -->
  
  <link rel="stylesheet" href="https://examplesite.com/scss/style.min.css" integrity="" media="screen">

  <!--Favicon-->
  <link rel="shortcut icon" href="https://examplesite.com/images/favicon.png " type="image/x-icon">
  <link rel="icon" href="https://examplesite.com/images/favicon.png " type="image/x-icon">

</head><body>
<!-- preloader start -->
<div class="preloader">
  
</div>
<!-- preloader end -->
<nav class="navbar navbar-expand-lg navbar-light bg-white w-100 navigation" id="navbar">
  <div class="container">
    <a class="navbar-brand" href="https://examplesite.com"><img class="img-fluid" src="https://examplesite.com/images/logo.png"
        alt="Adrian Hugo | E-commerce template"></a>

    <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#main-navbar"
      aria-controls="main-navbar" aria-expanded="false" aria-label="Toggle navigation">
      <span class="navbar-toggler-icon"></span>
    </button>

    <div class="collapse navbar-collapse " id="main-navbar">
      <ul class="navbar-nav mx-auto">
        <li class="nav-item">
          <a class="nav-link" href="https://examplesite.com"> Home </a>
        </li>
        
        
        <li class="nav-item">
          <a class="nav-link" href="https://examplesite.com/shop">Shop</a>
        </li>
        
        
        
        <li class="nav-item">
          <a class="nav-link" href="https://examplesite.com/about">About</a>
        </li>
        
        
        
        <li class="nav-item">
          <a class="nav-link" href="https://examplesite.com/faq">Faq</a>
        </li>
        
        
        
        <li class="nav-item">
          <a class="nav-link" href="https://examplesite.com/contact">Contact</a>
        </li>
        
        
      </ul>
    </div>
    

    <ul class="top-menu list-inline mb-0 d-none d-lg-block" id="top-menu">
      <li class="list-inline-item">
        <a href="#" class="search_toggle" id="search-icon"><i class="ti-search"></i></a>
      </li>
      <li class="dropdown cart-nav dropdown-slide list-inline-item">
        <a class="snipcart-checkout"><i class="ti-shopping-cart"></i></a>
      </li>
    </ul>
  </div>
</nav>


<div class="search-wrap">
  <div class="overlay">
    <form action="https://examplesite.com/search" class="search-form">
      <div class="container">
        <div class="row">
          <div class="col-md-10 col-9">
            <input type="search" id="search-query" name="s" class="form-control" placeholder="Search..." />
            <button type="submit" hidden></button>
          </div>
          <div class="col-md-2 col-3 text-right">
            <div class="search_toggle toggle-wrap d-inline-block">
              <img class="search-close" src="/images/close.png" alt="X" />
            </div>
          </div>
        </div>
      </div>
    </form>
  </div>
</div>


<section class="page-header" style="background-image: url('/');">
  <div class="overly"></div>
  <div class="container">
    <div class="row justify-content-center">
      <div class="col-lg-6">
        <div class="content text-center">
          <h1 class="mb-3">red</h1>
        </div>
      </div>
    </div>
  </div>
</section>

<section class="products-shop section">
  <div class="container">
    <div class="row">
      <div class="col-md-9">
        <div class="row">
          
          
          <div class="col-lg-4 col-12 col-md-6 col-sm-6 mb-5">
            <div class="product">
              <div class="product-wrap">
                <a href="https://examplesite.com/shop/black-tops/">
                  
                  <img class="img-fluid w-100 mb-3 img-first" src="/images/products/tops-1.jpg" />
                  
                  
                  
                  <img class="img-fluid w-100 mb-3 img-second" src="/images/products/tops-2.jpg" />
                  
                </a>
              </div>
              
              <div class="product-hover-overlay">
                <button class="snipcart-add-item"
                  data-item-id="black-tops__30.00"
                  data-item-name="Black Tops"
                  data-item-image="  https://examplesite.com/images/products/tops-1.jpg"
                  data-item-price="30.00"
                  data-item-url="https://examplesite.com/shop/black-tops/"
                  data-item-description="Nice and classy look tops for every girl."
                  
                  data-item-custom1-name="Choose Color"
                  data-item-custom1-options="Black|Red|Green"
                  
                  
                  data-item-custom2-name="Choose Size"
                  data-item-custom2-options="S|M|XL"
                  ><i class="ti-shopping-cart-full"></i></button>
              </div>

              <div class="product-info">
                <h2 class="product-title h5 mb-0"><a href="https://examplesite.com/shop/black-tops/">Black Tops</a></h2>
                <span class="price">
                  $30.00
                </span>
              </div>
            </div>
          </div>
          
          <div class="col-lg-4 col-12 col-md-6 col-sm-6 mb-5">
            <div class="product">
              <div class="product-wrap">
                <a href="https://examplesite.com/shop/floral-kirby/">
                  
                  <img class="img-fluid w-100 mb-3 img-first" src="/images/products/tops-3.jpg" />
                  
                  
                  
                  <img class="img-fluid w-100 mb-3 img-second" src="/images/products/tops-2.jpg" />
                  
                  
                  
                  
                </a>
              </div>
              
              <span class="onsale">Sale</span>
              
              <div class="product-hover-overlay">
                <button class="snipcart-add-item"
                  data-item-id="floral-kirby__20.00"
                  data-item-name="Floral Kirby"
                  data-item-image="  https://examplesite.com/images/products/tops-3.jpg"
                  data-item-price="20.00"
                  data-item-url="https://examplesite.com/shop/floral-kirby/"
                  data-item-description="Nice and classy look tops for every girl."
                  
                  data-item-custom1-name="Choose Color"
                  data-item-custom1-options="Blue|Red|Green"
                  
                  
                  data-item-custom2-name="Choose Size"
                  data-item-custom2-options="S|M|XL"
                  ><i class="ti-shopping-cart-full"></i></button>
              </div>

              <div class="product-info">
                <h2 class="product-title h5 mb-0"><a href="https://examplesite.com/shop/floral-kirby/">Floral Kirby</a></h2>
                <span class="price">
                  $20.00
                </span>
              </div>
            </div>
          </div>
          
          <div class="col-lg-4 col-12 col-md-6 col-sm-6 mb-5">
            <div class="product">
              <div class="product-wrap">
                <a href="https://examplesite.com/shop/purse/">
                  
                  <img class="img-fluid w-100 mb-3 img-first" src="/images/products/bag-4.jpg" />
                  
                  
                </a>
              </div>
              
              <div class="product-hover-overlay">
                <button class="snipcart-add-item"
                  data-item-id="purse__98.00"
                  data-item-name="Purse"
                  data-item-image="  https://examplesite.com/images/products/bag-4.jpg"
                  data-item-price="98.00"
                  data-item-url="https://examplesite.com/shop/purse/"
                  data-item-description="Hand bag for multi purpose use."
                  
                  data-item-custom1-name="Choose Color"
                  data-item-custom1-options="Red|Black"
                  
                  
                  data-item-custom2-name="Choose Size"
                  data-item-custom2-options="S"
                  ><i class="ti-shopping-cart-full"></i></button>
              </div>

              <div class="product-info">
                <h2 class="product-title h5 mb-0"><a href="https://examplesite.com/shop/purse/">Purse</a></h2>
                <span class="price">
                  $98.00
                </span>
              </div>
            </div>
          </div>
          
          <div class="col-12">
            


          </div>
        </div>
      </div>
      <div class="col-md-3">
        <section class="widget widget-categories mb-5">
          <h3 class="widget-title h4 mb-4">Categories</h3>
          <ul class="list-unstyled">
            <li>
              <a class="cat-item d-lg-flex justify-content-between text-dark"
                href="/categories/bag">Bag<span>4</span></a>
            </li>
            <li>
              <a class="cat-item d-lg-flex justify-content-between text-dark"
                href="/categories/shirt">Shirt<span>1</span></a>
            </li>
            <li>
              <a class="cat-item d-lg-flex justify-content-between text-dark"
                href="/categories/shoe">Shoe<span>4</span></a>
            </li>
            <li>
              <a class="cat-item d-lg-flex justify-content-between text-dark"
                href="/categories/tops">Tops<span>2</span></a>
            </li>
            <li>
              <a class="cat-item d-lg-flex justify-content-between text-dark"
                href="/categories/watch">Watch<span>4</span></a>
            </li>
          </ul>
        </section>
        <section class="widget widget-categories mb-5">
          <h3 class="widget-title h4 mb-4">Colors</h3>
          <ul class="list-unstyled">
            <li>
              <a class="cat-item d-lg-flex justify-content-between text-dark"
                href="/colors/black">Black<span>13</span></a>
            </li>
            
            <li>
              <a class="cat-item d-lg-flex justify-content-between text-dark"
                href="/colors/blue">Blue<span>3</span></a>
            </li>
            
            <li>
              <a class="cat-item d-lg-flex justify-content-between text-dark"
                href="/colors/golden">Golden<span>1</span></a>
            </li>
            
            <li>
              <a class="cat-item d-lg-flex justify-content-between text-dark"
                href="/colors/green">Green<span>2</span></a>
            </li>
            
            <li>
              <a class="cat-item d-lg-flex justify-content-between text-dark"
                href="/colors/pink">Pink<span>1</span></a>
            </li>
            
            <li>
              <a class="cat-item d-lg-flex justify-content-between text-dark"
                href="/colors/red">Red<span>3</span></a>
            </li>
            
            <li>
              <a class="cat-item d-lg-flex justify-content-between text-dark"
                href="/colors/white">White<span>8</span></a>
            </li>
            
          </ul>
        </section>
        <section class="widget widget-categories mb-5">
          <h3 class="widget-title h4 mb-4">Sizes</h3>
          <ul class="list-unstyled">
            <li>
              <a class="cat-item d-lg-flex justify-content-between text-dark"
                href="/sizes/m">M<span>10</span></a>
            </li>
            <li>
              <a class="cat-item d-lg-flex justify-content-between text-dark"
                href="/sizes/s">S<span>11</span></a>
            </li>
            <li>
              <a class="cat-item d-lg-flex justify-content-between text-dark"
                href="/sizes/xl">XL<span>7</span></a>
            </li>
            <li>
              <a class="cat-item d-lg-flex justify-content-between text-dark"
                href="/sizes/xxl">XXL<span>1</span></a>
            </li>
          </ul>
        </section>
      </div>
    </div>
  </div>
</section>

<footer class="footer">
  <div class="container">
    <div class="row">
      <div class="col-md-6 col-lg-4 col-sm-6 mb-5 mb-lg-0 text-center text-sm-left mr-auto">
        <div class="footer-widget">
          <a class="navbar-brand" href="https://examplesite.com"><img class="img-fluid" src="https://examplesite.com/images/logo.png"
          alt="Adrian Hugo | E-commerce template"></a>
          <p class="lead">Iste dolores iure quis excepturi, deserunt praesentium.</p>
          <p class="mb-0"><strong>Location : </strong>Dhaka, Bangladedsh</p>
          <p><strong>Support Email : </strong>demo@email.com</p>
        </div>
      </div>

      <div class="col-md-6 col-lg-2 col-sm-6 mb-5 mb-lg-0 text-center text-sm-left">
        <div class="footer-widget">
          <h4 class="mb-4">Category</h4>
          <ul class="pl-0 list-unstyled mb-0">
            <li><a href="/categories/bag">Bag</a></li>
            <li><a href="/categories/shirt">Shirt</a></li>
            <li><a href="/categories/shoe">Shoe</a></li>
            <li><a href="/categories/tops">Tops</a></li>
            <li><a href="/categories/watch">Watch</a></li>
          </ul>
        </div>
      </div>

      <div class="col-md-6 col-lg-2 col-sm-6 mb-5 mb-lg-0 text-center text-sm-left">
        <div class="footer-widget">
          <h4 class="mb-4">Useful Link</h4>
          <ul class="pl-0 list-unstyled mb-0">
            
            <li><a href="https://examplesite.com/shop">Shop</a></li>
            
            <li><a href="https://examplesite.com/about">About</a></li>
            
            <li><a href="https://examplesite.com/faq">Faq</a></li>
            
            <li><a href="https://examplesite.com/contact">Contact</a></li>
            
          </ul>
        </div>
      </div>

      <div class="col-md-6 col-lg-3 col-sm-6 text-center text-sm-left">
        <div class="footer-widget">
          <h4 class="mb-4">Opening Hours</h4>
          <ul class="pl-0 list-unstyled mb-5">
            
            <li class="d-lg-flex justify-content-between">Monday-Friday <span>8.00-20.00</span></li>
            
            <li class="d-lg-flex justify-content-between">Saturday <span>10.00-20.00</span></li>
            
            <li class="d-lg-flex justify-content-between">Sunday <span>12.00-20.00</span></li>
            
          </ul>

          <h5>Call Now : 0124857985320</h5>
        </div>
      </div>
    </div>
  </div>
</footer>


<div class="footer-btm py-4 ">
  <div class="container">
    <div class="row ">
      <div class="col-lg-6">
        <p class="copyright mb-0 ">@ Copyright Reserved to Adrian &amp; made by <a href="#">Themefisher</a></p>
      </div>
      <div class="col-lg-6">
        <ul class="list-inline mb-0 footer-btm-links text-lg-right mt-2 mt-lg-0">
          <li class="list-inline-item"><a href="404.html">Privacy Policy</a></li>
          <li class="list-inline-item"><a href="404.html">Terms &amp; Conditions</a></li>
          <li class="list-inline-item"><a href="404.html">Cookie Policy</a></li>
          <li class="list-inline-item"><a href="404.html">Terms of Sale</a></li>
        </ul>
      </div>
    </div>
  </div>
</div>




<script>
  var indexURL = "https://examplesite.com/index.json"
</script>



<div hidden id="snipcart" data-api-key="ZDgxY2ZiM2EtMWRmMi00M2ZkLWI1MDAtNzA2MDc0Y2Q1MzJlNjM3MTAzNDIwOTQ1NDA4NjM4"></div>


<script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyCcABaamniA6OL5YvYSpB3pFMNrXwXnLwU&amp;libraries=places"></script>


<script src="https://examplesite.com/plugins/jQuery/jquery.min.js"></script>

<script src="https://examplesite.com/plugins/bootstrap/bootstrap.min.js"></script>

<script src="https://examplesite.com/plugins/SyoTimer/jquery.syotimer.min.js"></script>

<script src="https://examplesite.com/plugins/slick/slick.min.js"></script>

<script src="https://cdn.snipcart.com/themes/v3.0.3/default/snipcart.js"></script>

<script src="https://examplesite.com/plugins/instafeed-js/instafeed.min.js"></script>

<script src="https://examplesite.com/plugins/search/fuse.min.js"></script>

<script src="https://examplesite.com/plugins/search/mark.js"></script>

<script src="https://examplesite.com/plugins/search/search.js"></script>

<script src="https://examplesite.com/plugins/google-map/gmap.js"></script>



<script src="https://examplesite.com/js/script.min.js"></script>

<script>
  (function (i, s, o, g, r, a, m) {
    i['GoogleAnalyticsObject'] = r;
    i[r] = i[r] || function () {
      (i[r].q = i[r].q || []).push(arguments)
    }, i[r].l = 1 * new Date();
    a = s.createElement(o),
      m = s.getElementsByTagName(o)[0];
    a.async = 1;
    a.src = g;
    m.parentNode.insertBefore(a, m)
  })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
  ga('create', 'Your ID', 'auto');
  ga('send', 'pageview');
</script></body>
</html>