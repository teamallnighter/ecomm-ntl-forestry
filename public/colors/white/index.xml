<?xml version="1.0" encoding="utf-8" standalone="yes"?>
<rss version="2.0" xmlns:atom="http://www.w3.org/2005/Atom">
  <channel>
    <title>white on Adrian Hugo | E-commerce template</title>
    <link>https://examplesite.com/colors/white/</link>
    <description>Recent content in white on Adrian Hugo | E-commerce template</description>
    <generator>Hugo -- gohugo.io</generator>
    <language>en-us</language>
    <lastBuildDate>Mon, 25 Nov 2019 13:39:07 +0600</lastBuildDate>
    
	<atom:link href="https://examplesite.com/colors/white/index.xml" rel="self" type="application/rss+xml" />
    
    
    <item>
      <title>Classic Watch</title>
      <link>https://examplesite.com/shop/classic-watch/</link>
      <pubDate>Mon, 25 Nov 2019 13:39:07 +0600</pubDate>
      
      <guid>https://examplesite.com/shop/classic-watch/</guid>
      <description>Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas. Vestibulum tortor quam, feugiat vitae, ultricies eget, tempor sit amet, ante.</description>
    </item>
    
    <item>
      <title>Female Bag</title>
      <link>https://examplesite.com/shop/female-bag/</link>
      <pubDate>Mon, 25 Nov 2019 13:39:07 +0600</pubDate>
      
      <guid>https://examplesite.com/shop/female-bag/</guid>
      <description>Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas. Vestibulum tortor quam, feugiat vitae, ultricies eget, tempor sit amet, ante.</description>
    </item>
    
    <item>
      <title>Hand Bag</title>
      <link>https://examplesite.com/shop/hand-bag/</link>
      <pubDate>Mon, 25 Nov 2019 13:39:07 +0600</pubDate>
      
      <guid>https://examplesite.com/shop/hand-bag/</guid>
      <description>Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas. Vestibulum tortor quam, feugiat vitae, ultricies eget, tempor sit amet, ante.</description>
    </item>
    
    <item>
      <title>Hand Watch</title>
      <link>https://examplesite.com/shop/hand-watch/</link>
      <pubDate>Mon, 25 Nov 2019 13:39:07 +0600</pubDate>
      
      <guid>https://examplesite.com/shop/hand-watch/</guid>
      <description>Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas. Vestibulum tortor quam, feugiat vitae, ultricies eget, tempor sit amet, ante.</description>
    </item>
    
    <item>
      <title>Keds</title>
      <link>https://examplesite.com/shop/keds/</link>
      <pubDate>Mon, 25 Nov 2019 13:39:07 +0600</pubDate>
      
      <guid>https://examplesite.com/shop/keds/</guid>
      <description>Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas. Vestibulum tortor quam, feugiat vitae, ultricies eget, tempor sit amet, ante.</description>
    </item>
    
    <item>
      <title>Orange Bag</title>
      <link>https://examplesite.com/shop/orange-bag/</link>
      <pubDate>Mon, 25 Nov 2019 13:39:07 +0600</pubDate>
      
      <guid>https://examplesite.com/shop/orange-bag/</guid>
      <description>Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas. Vestibulum tortor quam, feugiat vitae, ultricies eget, tempor sit amet, ante.</description>
    </item>
    
    <item>
      <title>Smart Watch</title>
      <link>https://examplesite.com/shop/smart-watch/</link>
      <pubDate>Mon, 25 Nov 2019 13:39:07 +0600</pubDate>
      
      <guid>https://examplesite.com/shop/smart-watch/</guid>
      <description>Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas. Vestibulum tortor quam, feugiat vitae, ultricies eget, tempor sit amet, ante.</description>
    </item>
    
    <item>
      <title>Sneakers</title>
      <link>https://examplesite.com/shop/sneakers/</link>
      <pubDate>Mon, 25 Nov 2019 13:39:07 +0600</pubDate>
      
      <guid>https://examplesite.com/shop/sneakers/</guid>
      <description>Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas. Vestibulum tortor quam, feugiat vitae, ultricies eget, tempor sit amet, ante.</description>
    </item>
    
  </channel>
</rss>