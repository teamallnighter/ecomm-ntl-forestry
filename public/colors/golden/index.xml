<?xml version="1.0" encoding="utf-8" standalone="yes"?>
<rss version="2.0" xmlns:atom="http://www.w3.org/2005/Atom">
  <channel>
    <title>golden on Adrian Hugo | E-commerce template</title>
    <link>https://examplesite.com/colors/golden/</link>
    <description>Recent content in golden on Adrian Hugo | E-commerce template</description>
    <generator>Hugo -- gohugo.io</generator>
    <language>en-us</language>
    <lastBuildDate>Mon, 25 Nov 2019 13:39:07 +0600</lastBuildDate>
    
	<atom:link href="https://examplesite.com/colors/golden/index.xml" rel="self" type="application/rss+xml" />
    
    
    <item>
      <title>Lady Shoe</title>
      <link>https://examplesite.com/shop/lady-lofer/</link>
      <pubDate>Mon, 25 Nov 2019 13:39:07 +0600</pubDate>
      
      <guid>https://examplesite.com/shop/lady-lofer/</guid>
      <description>Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas. Vestibulum tortor quam, feugiat vitae, ultricies eget, tempor sit amet, ante.</description>
    </item>
    
  </channel>
</rss>